### PR TITLE
Thunder R4.4 Build Error Fixed when ProcessContainer enabled

### PR DESCRIPTION
**Pull request by tabbas651**
* Fixed Thunder R4.4..1 Compilation error when enabled ProcessContiainer 
* Correct the UnLock spelling
* Correct the messaging  header file
* updated the ProcessContainers::ProcessContainerization into Trace::Information

### CODE REVIEW

**Review by karuna2git — APPROVED** [review]

File: Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp [hunk 1]
```diff
@@ -381,10 +381,10 @@ namespace ProcessContainers {
             TRACE(Trace::Error, (_T("Failed to start container %s - internal Dobby error."), _name.c_str()));
             result = false;
         } else {
-            TRACE(ProcessContainers::ProcessContainerization, (_T("started %s container! descriptor: %d"), _name.c_str(), _descriptor));
+            TRACE(Trace::Information, (_T("started %s container! descriptor: %d"), _name.c_str(), _descriptor));
             result = true;
         }
-        _adminLock.UnLock();
+        _adminLock.Unlock();
```
Comment (karuna2git) [inline Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp:387]: ProcessContainers were never compiled for opensource community. So this was not noticed. Thanks for pointing it out

File: Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.h [hunk 2]
```diff
@@ -24,7 +24,7 @@
 #include "processcontainers/common/BaseRefCount.h"
 #include "processcontainers/common/CGroupContainerInfo.h"
 
-#include "Messaging.h"
+#include "messaging/messaging.h"
```
Comment (karuna2git) [inline Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.h:27]: ProcessContainers were never compiled for opensource community. So this was not noticed

File: Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp [hunk 3]
```diff
@@ -361,7 +361,7 @@ namespace ProcessContainers {
         }
 
         if (_useSpecFile) {
-            TRACE(ProcessContainers::ProcessContainerization, (_T("path set to %s for name %s"), _path.c_str(), _name.c_str()));
+            TRACE(Trace::Information, (_T("path set to %s for name %s"), _path.c_str(), _name.c_str()));
```
Comment (karuna2git) [inline Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp:364]: changing everything from `ProcessContainers::ProcessContainerization` to `Trace::Information` is okey for now.

**Review by karuna2git — APPROVED** [review]

File: Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp [hunk 4]
```diff
@@ -56,16 +56,16 @@ namespace ProcessContainers {
             };
 
             Core::File configBundleFile(path + "/Container" + CONFIG_NAME);
-            TRACE(ProcessContainers::ProcessContainerization, (_T("searching %s container at %s, %s"), id.c_str(), configBundleFile.Name().c_str(), configBundleFile.PathName().c_str()));
```
Comment (MFransen69) [inline Source/extensions/processcontainers/implementations/DobbyImplementation/DobbyImplementation.cpp:59]: Please also see our comments in the PR for master

**Review by MFransen69 — CHANGES_REQUESTED** [review]

**Comment by MFransen69** [comment]
We agreed this PR to be closed and the fixes to be incorporated into a fix for the remaining process containers issues

**Pull request closed without merging**